python/private/py_wheel.bzl

```
@@ -117,6 +118,21 @@ See [`py_wheel_dist`](/docs/packaging.md#py_wheel_dist) for more info.
     ),
 }
 
+_feature_flags = {
+    "incompatible_normalize_name": attr.bool(
+        default = False,
+        doc = "Normalize the package distribution name according to current " +
+              "standards.",
```

nit:
```suggestion
        doc = """\
Normalize the package distribution name according to latest
Python packaging standards.

See https://packaging.python.org/en/latest/specifications/binary-distribution-format/#escaping-and-unicode
and https://packaging.python.org/en/latest/specifications/name-normalization/.

Apart from the valid names according to the above, we also accept
'{' and '}', which may be used as placeholders for stamping.
""",
```

Later this doc string could be moved to the documentation of the `attr.distribution` documentation. Having the links to the official Python documentation may be nice.